sqlmesh/core/snapshot/evaluator.py

```diff
@@ -958,10 +964,13 @@ def append(
 
     def create(
         self,
-        model: Model,
+        snapshot: Snapshot,
         name: str,
         **render_kwargs: t.Any,
     ) -> None:
+        model = snapshot.model
+        self.adapter.execute(model.render_pre_statements(**render_kwargs))
```

Comment:
That's what I'm talking about. What's the point of moving pre / post statements into strategies?